dataland-frontend/src/model/Data.js

```diff
@@ -0,0 +1,19 @@
+export class Data {
+    /* The fields of this class and APIClient.vue will be changed */
```

Comment:
Use JSDocs: 
/**
*
*/ 
instead of comment

Reply:
getan für alle js Dateien